modules/apps/static/portal-configuration/portal-configuration-persistence/src/main/java/com/liferay/portal/configuration/persistence/ConfigurationPersistenceManager.java

```diff
@@ -316,6 +333,16 @@ protected void doStore(
 			String pid, @SuppressWarnings("rawtypes") Dictionary dictionary)
 		throws IOException {
 
+		ConfigurationModelListener configurationModelListener = null;
+
+		if ((_serviceTrackerMap != null) && hasPid(pid)) {
```

Comment:
Output of failure: https://gist.github.com/drewbrokke/c94a744ad54fb2a156b67e6c34847b2d